rotkehlchen/db/eth2.py

```diff
@@ -192,3 +193,27 @@ def delete_validator(self, validator_index: Optional[int], public_key: Optional[
                     f'Tried to delete eth2 validator with {field} '
                     f'{input_tuple[0]} from the DB but it did not exist',
                 )
+
+    def get_validators_profit(
+            self,
+            withdrawals_filter_query: EthStakingEventFilterQuery,
+            execution_filter_query: EthStakingEventFilterQuery,
+    ) -> list[FVal]:
```

Comment:
As said in the comment below, I think it will be better to be explicit here and just return a tuple of 2 values.

In the place where this function is used, it is used in format `withdrawn_amount, execution_layer_rewards = dbeth2.get_validators_profit(`, so the values are unpacked anyway.